### DISCUSSION TITLE
page with overall projects tasks status bottlenecks

### DISCUSSION THREAD

**Discussion by Stormfinger** [Leantime/leantime#737] [Ideas]
hi
i have around 50 projects in Leantime. im one man with lots of ideas. it would be nice if there was overall more convinient ALL-projects page.

**Comment by marcelfolaron** [comment]
That’s amazing. Would love to hear your thoughts and feedback!

**Comment by gloriafolaron** [comment]
@Stormfinger 50 projects.  I’d be super curious to hear more on how you’re using the system.

any thoughts on what an all-projects view might look like? Thinking just a list thrown up there or something more high level relating to a status you want to see quickly?  Or maybe more of a way to sort/organize? 

Great call out, btw. Thanks for contributing.

**Comment by Stormfinger** [comment]
hi
well one thing i like that there are web hooks ..im looking for free time to study it and connect leanproject to NextCloud ..couse my docs are all over the place copied ..im falling behind on schedule with organizing projects.. one thing i already mentioned is cost of a project. also what i did find anoying a bit is when i started i had to create a project to enter leantime ...i think its great setup and job well done when managing one project .. but what im missing you know a morning coffee set up like overview what will i be doing or must do today.. i will try to provide additional info  ...something like im stuck report or projects pending because of issue i dont know if there is anything like that.. im writing this without currently providing proper solution ..only because im strugling with time ..

**Reply by gloriafolaron** [reply]
Thanks for the feedback here!  We’re working on some designs for a better “morning coffee” project custom view.

Thinking of it like a newspaper, are there any particular headlines that you’d want to see with your coffee?

**Comment by Stormfinger** [comment]
hi 
yes i like the idea of newspaper style.. perhaps with two pages or more.. one page with latest finished tasks and one page with next to do, perhaps third page with statistics. I havent yet got real filling with ideas within leantime...but yes daily morning newspaper with sorta like articles is great idea. i do apologize for late response.
